reactioncommerce/kinetic
# [MCOSS-42] Polish sidebar UI and implement System information

Changes:
- Logic changes LOC around 400 LOCs, other lines come from graphql generated file
- Polish sidebar component UI to align with new design
- Implement system information modal

Testing instruction:
- Start API server and application
- Login with a valid account
- New sidebar UI should appear
![Screen Shot 2022-07-11 at 11 38 50](https://user-images.githubusercontent.com/33818318/178189625-e101ce96-b074-434a-8ba9-2cb07700ddeb.png)
![Screen Shot 2022-07-11 at 11 38 43](https://user-images.githubusercontent.com/33818318/178189636-8f65b158-7eec-42cb-bef3-e50f6fd2db25.png)

- Click on System Information
![image](https://user-images.githubusercontent.com/33818318/178189693-2ca413b9-4de8-4cbb-b3a0-07e0b5ca3dae.png)
